client/task-list/tasks/payments/paypal.js

```diff
@@ -167,200 +246,197 @@ export class PayPal extends Component {
 	}
 
 	getInitialConfigValues() {
-		return {
-			api_username: '',
-			api_password: '',
-			create_account: this.isWooCommerceServicesConnected(),
-			account_email: '',
-		};
+		const { options } = this.props;
+		return [
+			'merchant_email',
+			'merchant_id',
+			'client_id',
+			'client_secret',
+		].reduce( ( initialVals, key ) => {
+			return {
+				...initialVals,
+				[ key ]:
+					options && options[ key + '_production' ]
+						? options[ key + '_production' ]
+						: '',
+			};
+		}, {} );
 	}
 
 	validate( values ) {
 		const errors = {};
 
-		if ( ! values.create_account && ! values.api_username ) {
-			errors.api_username = __(
-				'Please enter your API username',
+		if ( ! values.merchant_email ) {
+			errors.merchant_email = __(
+				'Please enter your Merchant email',
 				'woocommerce-admin'
 			);
 		}
-		if ( ! values.create_account && ! values.api_password ) {
-			errors.api_password = __(
-				'Please enter your API password',
+		if ( ! isEmail( values.merchant_email ) ) {
+			errors.merchant_email = __(
+				'Please enter a valid email address',
 				'woocommerce-admin'
 			);
 		}
-
-		if (
-			this.isWooCommerceServicesConnected() &&
-			values.create_account &&
-			! isEmail( values.account_email )
-		) {
-			errors.account_email = __(
-				'Please enter a valid email address',
+		if ( ! values.merchant_id ) {
+			errors.merchant_id = __(
+				'Please enter your Merchand Id',
```

Comment:
Typo: 'Merchant'. Should probably be 'ID' too.

Reply:
Fixed in: https://github.com/woocommerce/woocommerce-admin/pull/6261/commits/73fc01e1916268425e74163e8fc86209fb03a332